app/guid-file/route.ts

```diff
@@ -33,6 +39,21 @@ export default class FileDetail extends Route.extend() {
         }
     }
 
+    async afterModel(this: FileDetail, model) {
+        const dateCreated = model.file.get('dateCreated');
+        const dateModified = model.file.get('dateModified');
+        const institutions = await model.file.get('user').get('institutions');
+        const metaTagsData: MetaTagsData = {
```

Comment:
Question on TS patterns in general: Is it important/worth it to type local variables before they're passed to typed functions? 'metaTagsData' will be checked against 'MetaTagsData' when it's passed to `getHeadTags`, anyway, so I'm not sure we get anything extra by importing the type and applying it here.

Reply:
Actually, you're right. When I originally wrote this I had forgotten to type the service import, so I wasn't getting type checking. Now that I have that, no need to set an explicit type.

Reply:
The one thing this does give you is type checking where the object is defined in addition to where it is passed to `getHeadTags()`, which could be a little easier to debug.

Reply:
Hmm, true. I'd be fine leaving this as a manner of personal taste, but I probably won't do it.

Reply:
It does add unneeded noise, though. I probably won't do it, either as I prefer cleaner code.